addons/OPT/dynmarkerverwaltung/fn_clientInit.sqf

```diff
@@ -0,0 +1,72 @@
+/**
+* Überwachung ob Dialog vorhanden ist 
+* 
+* Autor: [GNC]Lord-MDB
+*
+* Argumente:
+* keine
+*
+* Rückgabewert:
+* keine
+*
+* Server Only:
+* Nein
+* 
+* Lokal:
+* Ja
+* 
+* Global:
+* Nein
+* 
+* API:
+* nein
+* 
+* Beispiel Externer Aufruf:
+* [] call EFUNC(dynmarkerverwaltung,clientInit);
+* Beispiel interner Aufruf:
+* [] call FUNC(clientInit);
+*
+*/
+
+#include "macros.hpp"
+
+diag_log "Successfully loaded the OPT/dynmarkerverwaltung module on the client";
```

Comment:
Hier das `DUMP`-Makro verwenden: Also so: `DUMP("Successfully loaded the OPT/dynmarkerverwaltung module on the client");`